src/runtime/CMakeLists.txt

```diff
@@ -21,85 +23,103 @@ set(HIPSYCL_SOURCES
   hw_model/memcpy.cpp
   serialization/serialization.cpp)
 
+# syclcc already knows about these include directories, but clangd-based tooling does not.
+# Specifying them explicitly ensures that IDEs can resolve all hipSYCL includes correctly.
+target_include_directories(hipSYCL-rt
+  PUBLIC
+    $<BUILD_INTERFACE:${HIPSYCL_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>
+  PRIVATE
+    ${HIPSYCL_SOURCE_DIR}
+    ${PROJECT_BINARY_DIR}/include
+)
+
+install(TARGETS hipSYCL-rt
+        EXPORT install_exports
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib/static)
```

Comment:
don't really get the benefit of moving the archive into a separate folder (I know, it's not new... but I still don't get it)

Reply:
I don't actually know where this decision originally came from.. I just copy-pasted the existing pattern ;)

Reply:
I have removed the separate installation directory.